config/sync/config_split.config_split.devel_split.yml

```diff
@@ -0,0 +1,16 @@
+uuid: 8e2b108b-5ea7-428e-9ac6-70d77bc34471
+langcode: en
+status: false
+dependencies: {  }
+id: devel_split
+label: 'Devel Split'
+description: ''
+folder: ../config/splits/development
+module:
+  devel: 0
+theme: {  }
+blacklist: {  }
+graylist: {  }
+graylist_dependents: true
+graylist_skip_equal: true
+weight: 0
```

Comment:
This is and the other 2 files are in correct you're supposed to have one "split" called development and within that split you choose devel, field ui and views ui.

Reply:
Once it's done all 3 file should automatically get added to the development dir

Reply:
I can't select multiple modules in one split, as I have mentioned in notion.

Reply:
You can, you hold shift and click, that's why there's a task to install "chosen", it makes it easier on multi select elements

Reply:
Cool, thanks.